app/server/server.js

```diff
@@ -49,4 +68,45 @@ app.get('/inventory', async (req, res) => {
 
 app.get('/checkout', (req, res) => {
   res.send('Checkout');
+});
+
+app.post('/user', async (req, res) => {
+  const { email, status } = req.body;
+  let user;
+  const result = await execute('/sql/users/get_by_email.sql', [email, status]);
+
+  if (!result.rows.length) {
+    user = await execute('/sql/users/put.sql', [email, status]);
+  } else {
+    user = result.rows[0];
+  }
+});
+
+app.post("/session/login", async (req, res) => {
```

Comment:
quotation consistency


```suggestion
app.post('/session/login', async (req, res) => {
```